core/storage/question/gae_models.py

```diff
@@ -244,6 +246,32 @@ def get_question_skill_links_and_skill_descriptions(
         )
         return question_skill_link_models, skill_descriptions, next_cursor_str
 
+    @classmethod
+    def get_question_skill_links_with_constant_number_per_skill(
+            cls, question_count, skill_ids):
+        """Fetches the list of constant number of QuestionSkillLinkModels
+        linked to the skills.
+
+        Args:
+            question_count: int. The number of questions to be returned.
+            skill_ids: list(str). The ids of skills for which the linked
+                question ids are to be retrieved.
+
+        Returns:
+            list(QuestionSkillLinkModel). Constant number of
```

Comment:
This is not clear. Please specify how many items there are in the returned list and any other constraints. Consider cases where there are not enough questions (either for a skill or in total), etc.

Reply:
Done.